pom.xml

```diff
@@ -248,7 +248,8 @@
       <version.jcipannotations>1.0</version.jcipannotations>
       <version.jetty.http.client>9.4.5.v20170502</version.jetty.http.client>
       <version.jgoodies.forms>1.0.7</version.jgoodies.forms>
-      <version.jgroups>4.0.8.Final</version.jgroups>
+      <!-- Make sure you also update the BOM -->
```

Comment:
I'm not sure this comment is necessary as a dependencies version should also be updated in both the bom and parent if they are located in both. Maybe we need to add a comment explaining this above all of the version properties in the pom.

Reply:
The only reason to redeclare it in the parent is if we need to use the property aside from the dependecy declaration, otherwise we should avoid it.

Reply:
@ryanemerson I only added the comment because I was surprised to see some properties duplicated in the parent and in the BOM, I don't remember having to modify the version in multiple places before.

Reply:
@danberindei before it wasn't necessary as properties were inherited from the bom, however we now import the bom which does not automatically import properties hence we have the duplication. In the case of jgroups, it is necessary as we require the jgroups version [here](https://github.com/infinispan/infinispan/blob/master/all/embedded/pom.xml#L331)